common/test/acceptance/tests/video/test_studio_video_module.py

```diff
@@ -320,3 +319,31 @@ def test_slider_range_works(self):
         self.save_unit_settings()
 
         self.video.click_player_button('play')
+
+
+@attr('a11y')
+class CMSVideoA11yTest(CMSVideoBaseTest):
+    """
+    CMS Video Accessibility Test Class
+    """
+
+    def setUp(self):
+        browser = os.environ.get('SELENIUM_BROWSER', 'firefox')
+
+        # the a11y tests run in CI under phantomjs which doesn't
+        # support html5 video or flash player, so the video tests
+        # don't work in it. We still want to be able to run these
+        # tests in CI, so override the browser setting if it is
+        # phantomjs.
```

Comment:
That's how this works.  Alternatively,  I could run all of the accessibility tests under firefox.